material-methoden.tex

```diff
@@ -129,7 +129,7 @@ \section{Zahlen und Einheiten}
 Es empfiehlt sich, das Paket \textit{siunitx}\footnote{\url{http://ctan.mirror.garr.it/mirrors/CTAN/macros/latex/contrib/siunitx/siunitx.pdf}} für die Darstellung von Zahlen und zugehöriger Einheit zu verwenden. Damit lassen sich Zahlen und kompliziertere Einheiten übersichtlich darstellen, beispielsweise \SI{42}{\kilo\gram\metre\per\square\second}. Weiterhin ist zu beachten:
 
 \begin{compactitem}
-	\item einheitenbehaftete Zahlen werden immer mit der dazugehörigen Einheit genannt und geschrieben
+	\item einheitenbehaftete Zahlen werden immer mit der dazugehörigen Einheit genannt, geschrieben und nicht voneinander getrennt.
 	\item Einheiten werden nicht kursiv geschrieben
```

Comment:
Gute Ergänzung! Mit dem SI-Package ist das eh nicht möglich, glaube ich.

Reply:
Okay. Good to know.